src/sentry/conf/server.py

```diff
@@ -25,7 +25,11 @@ def gettext_noop(s):
 socket.setdefaulttimeout(5)
 
 
-def env(key, default="", type=None):
+def env(
+    key: str,
+    default: Union[str, int, float, bool, None] = "",
+    type: Optional[Callable[[Any], Any]] = None,
+):
```

Comment:
Should we type the response too?

Reply:
+1

Reply:
The return type would have to be Any :cry: We might be able to get better types with overloads though.